README.md

```diff
@@ -656,6 +657,7 @@ BEARER_TOKEN: A valid bearer token for the account to associate the payment with
 | Field                    | required | Description                               |
 | ------------------------ |:--------:| ----------------------------------------- |
 | `amount`                 | Yes      | Amount to refund in pence                 |
+| `refund_amount_available`| No       | Total amount still available for refunds  |
```

Comment:
@rahul-vinayak @heathd This has been addressed. The new description now reads:  "_Total amount still available before issuing the refund_"